src/modules/Laboratorio/Visualizar/index.js

```diff
@@ -1,42 +1,58 @@
-import React from 'react';
+import React, {Component} from 'react';
 
 import Menu from '../../../utils/components/Menu';
 import Title from '../../../utils/components/Title';
 import Toolbar from '../../../utils/components/Toolbar';
+import Equipamento from './components/equipment';
 
-import './styles/index.css';
+class Listar extends Component {
+  
+  constructor(props){
+    super(props);
+    this.state ={
+      isLoaded: false,
+      eqp: [],
+      eqpInLab: []  
+    }
+  }
 
-function Visualizar() {
-  return (
-    <div>
-      <Toolbar />
-      <Menu />
-      <Title title="Visualizar Laboratório" subTitle="Laboratório 01" />
-      <div className="labVisualizar">
-        <div className="labVisualizarTitle">
-          <h1> Laboratório 01 </h1>
-        </div>
-        <div className="labVisualizarInfo">
-          <div className="labVisualizarData">
-            <h1> Status: </h1>
-            <h2> Ocupado </h2>
-          </div>
-          <div className="labVisualizarData">
-            <h1> Capacidade: </h1>
-            <h2> 40 </h2>
-          </div>
-          <div className="labVisualizarData">
-            <h1> Localização: </h1>
-            <h2> DCOMP Antigo </h2>
-          </div>
-          <div className="labVisualizarData">
-            <h1> Chamados: </h1>
-            <h2> 2 </h2>
-          </div>
+  async componentDidMount(){
+    await fetch('https://comunicabackdev.herokuapp.com/equipment')
+    .then(res => res.json())  
+    .then(json => {
+        this.setState({
+          isLoaded: true,
+          eqp:json, 
+        })
+      })
+      for (let index = 0; index < this.state.eqp.length; index++) {
+        let element = this.state.eqp[index];
+        if (this.props.location.state.id === element.laboratory_id) {
+          this.setState({
+            eqpInLab: this.state.eqpInLab.concat(element)
+          })
+        }
+      }
+    }
+
+  render(){
+    return (
+      <div>
+        <Toolbar />
+        <Menu />
+        <Title title="Listar Equipamentos"/>
+          <div className="listaEquipamentos">
+            <ul>
+              {this.state.eqpInLab.map( item =>(
+                <li key = {item.id}>
+                    <Equipamento eqp = {item} path = {window.location.pathname}/>
+                </li>
+              ))}
```

Comment:
Esse trecho de código pode ser atribuído a uma variável antes do `return`, para tornar o código mais legível.

```suggestion
              {equipmentList}
```